backend/server.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+from controllers import file_controller
+from flask_cors import CORS
+import json
+import config
+
+app = Flask(__name__)
+api = Api(app)
+CORS(app)
+
+
+@app.route("/")
```

Comment:
Is this really needed? Each resource should be registered at its own route/endpoint.